src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const FEMALE = 'f';
+  let content = 'I am not married';
+
+  const partnerStatus = sex === FEMALE
+    ? 'husband'
+    : 'wife';
+
+  if (isMarried) {
+    content = `${partnerName} is my ${partnerStatus}`;
+  }
```

Comment:
Not critical, but you can improve this section
- use the ternary operator
- declare content as const
- change the name `content` to more descriptive `marriedStatusContent`, or similar

```suggestion

  const partnerStatus = sex === FEMALE
    ? 'husband'
    : 'wife';

  const marriedStatusContent = isMarried? `${partnerName} is my ${partnerStatus}` : 'I am not married';

```